src/main/resources/public/css/stylesheet.css

```diff
@@ -55,7 +45,35 @@ span {
     margin-right: 2em;
 }
 
-html, body {
-    height:100vh;
+html,
+body {
+    height: 100vh;
     color: #3b3d44;
-}
\ No newline at end of file
+}
+
+input, textarea {
+    width: 100%;
+    margin: 8px 0;
+}
+
+input[type=text], textarea{
+    padding: 12px 20px;
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+  }
+  
+  input[type=number]{
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+  }
+  
+  input[type=submit], input[type=reset] {
+    width: 49%;
+    padding: 14px 20px;
+    border: none;
```

Comment:
you found these looked better with no border?

Reply:
i'm not a front-end guy so it looked good to me

Reply:
yeah I was just wondering since the default styles of the buttons were being modified which I don't see too often